nomad/blocked_evals.go

```diff
@@ -47,6 +47,11 @@ type BlockedEvals struct {
 	// classes.
 	escaped map[string]wrappedEval
 
+	// system is the set of system evaluations that failed to start on nodes because of
+	// resource constraints. Retried on any change for those nodes
+	// map[node.ID][token]eval
```

Comment:
Is this type declaration needed?  If so, maybe elaborate how one should read it?
```suggestion
```